corehq/apps/hqwebapp/models.py

```diff
@@ -123,22 +123,28 @@ def pkce_required(client_id):
 
 
 class ServerLocation:
-    PRODUCTION = 'production'
     INDIA = 'india'
-    ENVS = (PRODUCTION, INDIA)
-
-    SUBDOMAINS = {
-        PRODUCTION: 'www',
-        INDIA: 'india',
-    }
-
-    CHOICES_DICT = {
-        PRODUCTION: (SUBDOMAINS[PRODUCTION], _("United States")),
-        INDIA: (SUBDOMAINS[INDIA], _("India")),
+    PRODUCTION = 'production'
+    ENVS = (INDIA, PRODUCTION)
+
+    DATA = {
```

Comment:
Nit: I'd prefer a more specific term than "data". "Data", "Util", etc often end up so generic that they don't make for particularly readable code. But `ServerLocation.DATA` might be obvious enough, or there might not be a better option.

Reply:
Would actually using `ENVS` or `ENVIRONMENTS` here be clearer? Each one is a set of loosely-related information about each environment, and it's the keys are themselves each environment names.